iroha/src/tx.rs

```diff
@@ -51,11 +51,13 @@ impl AcceptedTransaction {
         let current_time = SystemTime::now()
             .duration_since(SystemTime::UNIX_EPOCH)
             .expect("Failed to get System Time.");
-        (current_time - Duration::from_millis(self.payload.creation_time))
-            > min(
-                Duration::from_millis(self.payload.time_to_live_ms),
-                transaction_time_to_live,
-            )
+        let current_time = current_time - Duration::from_millis(self.payload.creation_time);
```

Comment:
Maybe use only one variable?

Reply:
Thought that it would help with readability